src/components/SearchPane/SearchPaneForm.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import Button from 'Button/Button';
+import Button from 'Button';
```

Comment:
Surely this still needs to be `Button/Button` ?

Reply:
Nope, can be either - Button has an index.ts file so this works... however it does beg the question "which way is better?" (as both methods are used in this repo 😫).

I would argue the above is better, as it means if the main export in Button changes (to e.g. ButtonView) we can just update the Button/index.ts file to export that, and not have to update every component which uses the Button component

Reply:
As discussed, if it's working in situ then all good. We can address the wider question of default vs named export in `src/index.ts` another time if need be

Reply:
Will fix conflicts and merge today